composer.json

```diff
@@ -1,38 +1,41 @@
 {
     "name": "polygon-io/api",
-    "description": "polygon.io php client",
+    "description": "polygon.io php-client",
+    "keywords": [
+        "openapitools",
+        "openapi-generator",
+        "openapi",
+        "php",
+        "sdk",
+        "rest",
+        "api"
+    ],
+    "homepage": "https://openapi-generator.tech",
     "type": "library",
     "license": "MIT",
     "authors": [
         {
-            "name": "Julien Prugne",
-            "email": "julien@webeleon.dev"
-        },
-        {
-            "name": "Ersin Demirtas",
-            "email": "ed5@me.com"
+            "name": "OpenAPI",
```

Comment:
It might make sense to have Polygon as the author so people know who to contact

Reply:
Yeah, that makes sense too. Will do. Thanks for checking this out.

Reply:
Fixed.